main.go

```diff
@@ -139,7 +139,7 @@ func main() {
 	collectorBuilder.WithKubeClient(kubeClient)
 
 	ksmMetricsRegistry := prometheus.NewRegistry()
-	ksmMetricsRegistry.Register(prometheus.NewProcessCollector(os.Getpid(), ""))
+	ksmMetricsRegistry.Register(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{}))
```

Comment:
As per the documentation, my understanding is that this diff is functionally equivalent.

Reply:
@tariq1890 Yes. It is the recommended usage according to the doc.